packages/host/tests/acceptance/code-mode-test.ts

```diff
@@ -293,7 +293,7 @@ module('Acceptance | code mode tests', function (hooks) {
     assert.dom('[data-test-directory="Person/"] .icon').hasClass('open');
   });
 
-  todo('recent file links are shown', async function (assert) {
+  skip('recent file links are shown', async function (assert) {
```

Comment:
how come we are skipping this?

Reply:
oh nevermind--i thought `todo()` was `test()`

Reply:
ya I have issue 5990 to bring this back, it passed for me locally but not in CI and I struggled for hours to get it working, then it caused some weirdness in this context because of the error thrown so I just made it be skipped entirely 😬